components/suggest/src/schema.rs

```diff
@@ -170,6 +169,22 @@ CREATE TABLE fakespot_custom_details(
     total_reviews INTEGER NOT NULL,
     FOREIGN KEY(suggestion_id) REFERENCES suggestions(id) ON DELETE CASCADE
 );
+CREATE VIRTUAL TABLE IF NOT EXISTS fakespot_fts USING FTS5(
+  title,
+  prefix='4 5 6 7 8 9 10 11',
+  content='',
```

Comment:
Per SQLite's [doc](https://www.sqlite.org/fts5.html#contentless_delete_tables), contentless-delete tables should be preferred over contentless table. Shall we enable that?

Reply:
Yes please, that makes the code much nicer.

Reply:
i think i missed the utility of the contentless-delete tables. looks much better